hooks/commands/build.sh

```diff
@@ -14,7 +13,7 @@ for line in $(plugin_read_list CACHE_FROM) ; do
 
   echo "~~~ :docker: Pulling cache image for $service_name"
   if retry "$pull_retries" plugin_prompt_and_run docker pull "$service_image" ; then
-    cache_from[$service_name]=$service_image
+    printf -v "cache_from__${service_name}" %s "$service_image"
```

Comment:
I had to look this up:

```
printf [-v var] format [arguments]

The -v option causes the output to be assigned to the variable var rather than being printed to the standard output.
```

Reply:
TIL :)

Reply:
this is my google-driven development at its best :D